docker-compose.yml

```diff
@@ -2,8 +2,7 @@ version: '3.8'
 
 services:
   postgres:
-    build:
-      dockerfile: Dockerfile
+    build: .
```

Comment:
@hectormachin It looks like this was a test change? Did it fix something? Do we know why we need this to change?